controller/volume_controller.go

```diff
@@ -876,6 +881,10 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 		v.Status.Robustness = types.VolumeRobustnessFaulted
 		v.Status.CurrentNodeID = ""
 
+		// Since all replica failed, mark engine controller salvage requested
+		e.Spec.SalvageRequested = true
```

Comment:
Don't we need to check `autoSalvage` before setting this field?

Reply:
Yes, good catch! Fix it. Thanks.